azure-pipelines.yml

```diff
@@ -65,7 +65,7 @@ stages:
         workspace:
           clean: all
         displayName: Build Windows Phase
-        timeoutInMinutes: 120
+        timeoutInMinutes: 60
```

Comment:
Occasionally the windows tasks just hang and they'll just hold up the server for 2 hours. The windows builds never take more than 40 minutes